Automattic/wp-calypso
# Remove unused stepper frameworks

#### Changes proposed in this Pull Request

* This removes our first attempts for creating stepper framework.

#### Testing instructions

1. Smoke test `/stepper` and make sure everything works.
2. Make sure tests are passing.
